### PR TITLE
Clean up `project.json` files

### PR DESCRIPTION
**Pull request by dougbu**
- avoid deprecation warnings
- use latest version of the SQL Client package
- remove some redundant dependencies

### CODE REVIEW

File: test/Microsoft.AspNetCore.Tests.Throughput/project.json [hunk 1]
```diff
@@ -20,5 +19,6 @@
         "netstandardapp1.5"
       ]
     }
-  }
+  },
+  "testRunner": "xunit"
```
Comment (dougbu) [inline test/Microsoft.AspNetCore.Tests.Throughput/project.json]: I should probably leave this out and let our builds continue to skip tests in this project. With this in the file, all tests in the project fail due to missing environment variables and assertions in the `BasicThroughputTests` constructor.

Or, should this project be killed with fire?

Comment (shhsu) [inline test/Microsoft.AspNetCore.Tests.Throughput/project.json]: I would suggest leave this out and create a new issue for it

Comment (dougbu) [inline test/Microsoft.AspNetCore.Tests.Throughput/project.json]: #162

**Comment by dougbu** [comment]
:up::date:

**Comment by pranavkm** [comment]
:shipit:

**Comment by shhsu** [comment]
:shipit:

**Comment by dougbu** [comment]
daa09ce

**Pull request closed without merging**